prow/spyglass/viewers/junit/junit_template.go

```diff
@@ -0,0 +1,121 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package junit
+
+const tmplt = `
```

Comment:
I continue to think that development on these will be a real chore. There is nothing nice about the UX of embedding one language in another. No syntax checks, no highlighting, etc. It's going to be really hard to review... can we build this more programattically, or add some functionality to check these for validity so local iteration is useful?

Reply:
This is how I chose to develop my version of the view, I agree that this isnt ideal for syntax highlighting, but the viewer implementations are not the focus here. Follow-up PRs are welcome to change the standard viewer development workflow to something more friendly